crypto/multisig/wire.go

```diff
@@ -0,0 +1,26 @@
+package multisig
+
+import (
+	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+)
+
+// TODO: Figure out API for others to either add their own pubkey types, or
+// to make verify / marshal accept a cdc.
+const (
+	ThresholdPubkeyAminoRoute = "tendermint/ThresholdMultisigPubkey"
+)
+
+var cdc = amino.NewCodec()
+
+func init() {
+	cdc.RegisterInterface((*crypto.PubKey)(nil), nil)
+	cdc.RegisterConcrete(ThresholdMultiSignaturePubKey{},
+		ThresholdPubkeyAminoRoute, nil)
+	cdc.RegisterConcrete(ed25519.PubKeyEd25519{},
+		ed25519.Ed25519PubKeyAminoRoute, nil)
```

Comment:
oh, should these just become`PubKeyAminoRoute` since it's an anti-pattern to have the double prefix (pkg and var name)?

Reply:
Addressed in https://github.com/tendermint/tendermint/pull/2205